test/e2e/util/utils.go

```diff
@@ -46,6 +50,8 @@ const (
 	FilterNodeSelector   = "kubernetes.io/hostname=eraser-e2e-test-worker"
 	FilterLabelKey       = "eraser.sh/cleanup.filter"
 	FilterLabelValue     = "true"
+	TestLogDir           = "../eraser_logs"
```

Comment:
This should be set in CI or something rather than a const.

So someone can do `make e2e LOGDIR=<path>`.
Can also set a default like `TEST_LOGDIR ?= $(PWD)/test_logs` or something in the Makefile.